src/index.js

```diff
@@ -127,6 +176,33 @@ export default function makeEPromiseClass(BasePromise) {
     }
   }
 
+  function makeForwarder(operation, localImpl) {
+    return async (ep, ...args) => {
+      const o = await ep;
+      const resolvedRelay = presenceToResolvedRelay.get(o);
+      if (resolvedRelay) {
+        // FIXME: Should we give a naked Presence to the relay?
```

Comment:
Since this is a resolvedRelay, yes.

Reply:
Thanks!  Comment adjusted in next commit.